build/args/all.gn

```diff
@@ -21,6 +21,4 @@ angle_enable_vulkan_validation_layers = false
 
 is_cfi = false
 
-# TODO: Remove this and update CI to contain 10.14 SDK once
-# crbug.com/986701 is fixed.
-mac_sdk_min = "10.13"
+mac_sdk_min = "10.14"
```

Comment:
This arg can be removed, its set to `10.14` in upstream, we were just working around it.